components/script/dom/element.rs

```diff
@@ -610,6 +610,8 @@ pub trait AttributeHandlers {
     /// name, if any.
     fn get_attribute(self, namespace: Namespace, local_name: &Atom)
                      -> Option<Temporary<Attr>>;
+    /// Returns the first attribute with given case-sensitive name, if any.
```

Comment:
nit: I'd prefer if the docs explicitly mentioned that it returns the first attr with a given case sensitive name and any namespace